.pipelines/.vsts-Marketpalce-win.yaml

```diff
@@ -2,7 +2,7 @@ name: $(Date:yyyyMMdd)$(Rev:.r)_$(Build.SourceBranchName)_$(BuildID)
 trigger: none
 
 pool:
-  vmImage: ubuntu-18.04
+  vmImage: ubuntu-24.04
```

Comment:
@timmy-wright can we migrate to self hosted 1ES pool if this pipeline is still being used?